src/rules/skins.ts

```diff
@@ -21,11 +21,19 @@ export const skinRules: RuleFn = (config) => {
     )
   );
 
+  const fillColors = makeRules(
```

Comment:
Cool, fyi this is going to be `makeRules` -> `newMethodsForProp` when you rebase.